.github/workflows/github_pr.yml

```
@@ -56,3 +56,45 @@ jobs:
         sudo apt-get install libsodium-dev
         sudo apt-get install g++-9
         ./ci/run.sh
+
+  windows_build:
+    name: Windows
+    strategy:
+      matrix:
+        os: [windows-2019]
+        dc: [dmd-2.090.1]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Prepare compiler
+      uses: mihails-strasuns/setup-dlang@v0.5.0
+      with:
+        compiler: ${{ matrix.dc }}
+        gh_token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Print compiler and dub's version
+      run: |
+        dmd --version
```

```suggestion
        ${DC} --version
```

Won't make a difference right now, but for the future.

---

```
Run ${DC} --version
At D:\a\_temp\cec021c4-9345-4f5c-bede-522e776aff25.ps1:2 char:9
+ ${DC} --version
+         ~~~~~~~
Unexpected token 'version' in expression or statement.
+ CategoryInfo          : ParserError: (:) [], ParseException
+ FullyQualifiedErrorId : UnexpectedToken
```